apps/backend/src/mcp/tools/proxy_management.py

```
@@ -0,0 +1,698 @@
+            query = select(ProxyConfig).options(selectinload(ProxyConfig.device))
+            
+            conditions = []
+            if device:
+                conditions.append(ProxyConfig.device_id == device)
+            if service_name:
+                conditions.append(ProxyConfig.service_name.ilike(f"%{service_name}%"))
+            if status:
+                conditions.append(ProxyConfig.status == status)
+            if ssl_enabled is not None:
+                if ssl_enabled:
+                    conditions.append(ProxyConfig.parsed_config['ssl_enabled'].astext.cast(bool) == True)
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `...:` for truth checks

Replace comparison

(E712)